compiler/src/dmd/expressionsem.d

```diff
@@ -41,6 +41,7 @@ import dmd.dstruct;
 import dmd.dsymbolsem;
 import dmd.dtemplate;
 import dmd.errors;
+import dmd.errorsink;
```

Comment:
Useless import

Reply:
yes, leftover from a previous incarnation of this PR